scripts/install

```diff
@@ -51,9 +50,7 @@ ynh_config_add_logrotate
 ynh_script_progression "Installing $app..."
 
 pushd $install_dir
-	ynh_hide_warnings ynh_exec_as_app yarn install
-	ynh_hide_warnings ynh_exec_as_app NODE_OPTIONS="--max-old-space-size=3000" yarn build:browser
-	ynh_hide_warnings ynh_exec_as_app yarn workspaces focus @actual-app/sync-server --production
+	ynh_hide_warnings npm install --location=global @actual-app/sync-server
```

Comment:
uuuuuuh does it really has to be installed in global ? That's usually a big no-no x_x